bin/oref0-mdt-update.sh

```diff
@@ -75,10 +78,7 @@ function show_last_record {
 # returns 0 if ture, 1 if false
 function glucose_fresh {
     if [ ! -f $FILE_MDT ]; then return 1; fi
-    # Enlite CGM are synced to the pump with a certain delay (not constant).
-    # Thus we shouldn't wait the whole 5m cycle but sync more often to get most recent values.
-    # I don't know what a good schedule is but I thing the Nyquist theorem is good starting point...
-    jq --exit-status "sort_by(.date) | reverse | .[0] | ($(date +%s) - .date / 1000) < 150" $FILE_MDT > /dev/null 
+    jq --exit-status "sort_by(.date) | reverse | .[0] | ($(date +%s) - .date / 1000) < 300" $FILE_MDT > /dev/null 
 }
```

Comment:
Is this change related?

Reply:
Well, my assumption from back then doesn't make sense at all. In practice on pi0 nothing changes by this change but on faster pis it will just sync when the bg is older than 5 minutes (instead of 2.5 minutes). To answer you question: no it's not related to the bug but it should be merged as well .. I changed it some time ago and pushed my most recent version.